packages/muelu/test/scaling/Driver.cpp

```diff
@@ -535,6 +541,12 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
 
   }//end reruns
 
+#ifdef HAVE_MUELU_AMGX
+// Finalize AMGX
+//MueLu::MueLu_AMGX_finalize();
+//MueLu::MueLu_AMGX_finalize_plugins();
+#endif
```

Comment:
Is there a reason you're not finalizing?

Reply:
Finalizing led to memory leak errors. I assume this is why the person who wrote the original MueLu_AMGXOperator_decl.hpp didn't include finalize either.